source/agora/common/DNS.d

```diff
@@ -399,6 +399,16 @@ public struct Header
     /// Reserved for future use. Must be zero in all queries and responses.
     public ubyte Z () const scope { return (this.field2 & 0b0_0000_0_0_0_0_111_0000) >> 4; }
```

Comment:
This needs an update I think,
https://datatracker.ietf.org/doc/html/rfc1035#section-4.1.1

Reply:
Fixed